pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -270,8 +273,9 @@ func (pl *PodTopologySpread) calPreFilterState(ctx context.Context, pod *v1.Pod,
 
 		tpCounts := make([]topologyCount, 0, len(constraints))
 		for i, c := range constraints {
-			if pl.enableNodeInclusionPolicyInPodTopologySpread &&
-				!c.matchNodeInclusionPolicies(pod, node, requiredNodeAffinity) {
+			// Ignore parsing errors for backwards compatibility.
```

Comment:
```suggestion
			// Error is a parse error, and we can just regard them as unmatched because that is how the tainttoleration plugin handles them. 
```